spec/requests/omniauth_callbacks_controller_spec.rb

```diff
@@ -246,6 +246,17 @@ def enabled?
         expect(data["destination_url"]).to eq(destination_url)
       end
 
+      it 'should return an associate url when multiple login methods enabled' do
+        get "/auth/google_oauth2/callback.json"
```

Comment:
```suggestion
        get "/auth/google_oauth2/callback.json"
        expect(response.status).to eq(302)
```

Asserting for the response first helps make it easier to debug if the test fails in the future.